src/XRP/network-clients/web-socket-network-client.ts

```diff
@@ -159,6 +165,53 @@ export default class WebSocketNetworkClient {
     return response as WebSocketStatusResponse
   }
 
+  /**
+   * Submits an account_lines request to the rippled Web Socket API.
+   * @see https://xrpl.org/account_lines.html
+   *
+   * @param account The XRPL account to query for trust lines.
+   */
+  public async getAccountLines(
+    account: string,
+    peerAccount?: string,
+  ): Promise<WebSocketAccountLinesResponse> {
+    const accountLinesRequest = {
+      id: 'account_lines_' + account,
+      command: 'account_lines',
+      account,
+      ledger_index: 'validated',
+      peer: peerAccount,
+    }
+    const accountLinesResponse = await this.sendApiRequest(accountLinesRequest)
+    return accountLinesResponse as WebSocketAccountLinesResponse
+  }
+
+  /**
+   * Submits a gateway_balances request to the rippled WebSocket API.
+   * @see https://xrpl.org/gateway_balances.html
+   *
+   * @param account The XRPL account for which to retrieve issued currency balances.
+   * @param addressesToExclude (Optional) An array of operational address to exclude from the balances issued.
+   * @see https://xrpl.org/issuing-and-operational-addresses.html
+   */
+  public async getGatewayBalances(
+    account: string,
+    addressesToExclude?: Array<string>,
+  ): Promise<WebSocketGatewayBalancesResponse> {
+    const gatewayBalancesRequest = {
+      id: 'gateway_balances_' + account,
+      command: 'gateway_balances',
+      account,
+      strict: 'true',
```

Comment:
should this be `true` instead of `'true'`? in `GatewayBalancesRequest` below, the type seems to suggest that this is a `boolean` instead of a string